src/main/java/se/swedenconnect/opensaml/pkcs11/credential/PKCS11Credential.java

```diff
@@ -173,6 +173,14 @@ private synchronized PrivateKey testPrivateKey(String providerName) {
         return currentPrivateKey;
     }
 
+    /**
+     * Get the provider of the most recently selected private key.
+     *
+     * <p>Note: This method is mainly intended for logging purposes. It may not be thread safe to rely on this method
+     * unless it is called from within a synchronized method after loading the private key.</p>
+     *
+     * @return THe name of the provider of the most recently loaded private key.
```

Comment:
**the** name ... (and no ending full stop)
Javadoc guide states that text following param and return should begin with a lower case letter and not end with a full stop.
Since we are generating javadoc ..